R/utils.R

```diff
@@ -1,11 +1,20 @@
 #'@importFrom httr stop_for_status GET user_agent content status_code
 pv_query <- function(params, reformat = TRUE, ...){
+  if(length(params) > 1){
+    data <- lapply(as.list(params), pv_query, reformat = reformat)
+    if(reformat == TRUE){
+      return(do.call(rbind, data))
+    } else {
+      return(data)
```

Comment:
Reformatting is not assumed...if reformatting is false, then a list of lists will be returned where each raw response is a list element. Each data frame uniquely defines what the views refer to, within the context of each api. As a result, it should not be an issue to rbind the results. :)